tests/sample/map_reuse_2.c

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// clang -O2 -Werror -c map_reuse_bad.c -o map_reuse_bad.o
+//
+// For bpf code: clang -target bpf -O2 -Werror -c map_reuse_bad.c -o map_reuse_bad.o
+// this passes the checker
+
+#include "bpf_helpers.h"
+#include "ebpf.h"
+
+#define PIN_NONE 0
+#define PIN_GLOBAL_NS 2
```

Comment:
These defines should not be needed since line 9 includes bpf_helpers.h which includes ebpf_structs.h which has an enum with these values.

Reply:
Removed.